docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
 The algorithm implemented is described in the article [Efficient Implementation of an Iterative Ensemble Smoother for Data Assimilation and Reservoir History Matching]( https://www.frontiersin.org/articles/10.3389/fams.2019.00047/full ).
 
 
+Kalman Posterior Properties
+---------------------------
 
+The updating from the prior :math:`p(\psi)=N\left(\mu_\psi,\Sigma_\psi\right)` 
+to the posterior :math:`p(\psi|d)=N\left(\mu_{\psi|d},\Sigma_{\psi|d}\right)`, assimilating measurements :math:`d`, 
+linear in :math:`\psi` is performed by the Kalman methods by employing the following equations
```

Comment:
What is "linear in :math:`\psi`"?

Reply:
the measurements `d`, so the comma should perhaps be after `psi`? Suggestion: it could be made more clear by: _... assimilating measurements `d` that are linear in `psi`, is performed..._

Reply:
That's better I think.